Document/Poll.php

```diff
@@ -77,6 +82,33 @@ public function setOptions(array $options = [])
     }
 
     /**
+     * @param $name
+     */
+    public function updateCount($name)
+    {
+        /** @var Option $option */
+        foreach ($this->options as $option) {
```

Comment:
`/** @var Option $option */` can be omitted when you changed the type-hint for the attribute $options.